src/components/caption/index.jsx

```diff
@@ -0,0 +1,27 @@
+import PropTypes from "prop-types";
+
+const COMPONENT_CLASS_NAME = "c-caption";
+
+const Caption = ({ caption, className, credit, title }) =>
+	!!(title || caption || credit) && (
+		<figcaption
+			className={className ? `${COMPONENT_CLASS_NAME} ${className}` : `${COMPONENT_CLASS_NAME}`}
+		>
+			{title && <span className="c-caption__title">{title} </span>}
+			{caption && <span className="c-caption__caption">{caption} </span>}
```

Comment:
Also note the requirement that these items can have HTML within them and will need to render that HTML

Reply:
Yep, thanks, included that in my todos